app/apps/show/operations/controller.js

```diff
@@ -1,5 +1,6 @@
 import Ember from 'ember';
+import Paginated from 'diesel/mixins/controllers/paginated';
```

Comment:
Agreed, although in some cases I find the absolute paths like these easier to use intra codebase. The relative paths are non-copy-pastable if the files are in different folder paths. I.e., to use Paginated in the apps/index route you'd `import Paginated from "../../mixins/routes/paginated"` but for the apps/operations/index it is `import Paginated from "../../../mixins/routes/paginated"`, while `import Paginated from "diesel/mixins/routes/paginated"` works for both. The volatility of import namespace (diesel) feels lower to me than volatility of folder nesting. But I am not religious about this.